tasks/translation/translate_content.py

```
@@ -0,0 +1,216 @@
+                    )
+                    translated = resp.choices[0].message.content.strip()
+                    # We don't get a confidence score from OpenAI; mark as 0.9
+                    return translated, 0.9
+                except Exception:
+                    return text, 0.0
+
+    except Exception:
+        OpenAIProvider = None
+
+
+    async def translate_content(
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)